src/plugins/intel_gna/gna_model_serial.cpp

```diff
@@ -535,6 +542,10 @@ void GNAModelSerial::Export(const GnaAllocations& allocations, std::ostream& os)
     for (const auto& a : allocationsOrdered) {
         os.write(reinterpret_cast<char*>(a.ptr), a.sizeForExport());
     }
+    
+    // write OV & GNA versions 
+    writeString(ov::intel_gna::common::get_openvino_version_string(), os);
+    writeString(GNADeviceHelper::GetGnaLibraryVersion(), os);    
```

Comment:
We should to consider to use TLV here

Reply:
Usage of TLV requires refactoring of gna2_model_export_helper.cpp, let use string serializing already provided in GNA model serialization module.

Reply:
OK